<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTextBrowser" name="NetlistLoadedFileInfo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="NetlistEditorLaunchWidget">
     <property name="text">
      <string>Launch Netlist Editor</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="NetlistDescriptionLoaderWidget">
     <property name="text">
      <string>Load Netlist Circuit Description</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="NetlistDynamicAdjustmentLoaderWidget">
     <property name="text">
      <string>Load Dynamic Adjustment Python Script</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
